packages/node_modules/samples/browser-meetings/index.html

```diff
@@ -29,18 +34,21 @@
     <h1>THIS IS A WORK IN PROGRESS SAMPLE</h1>
     <section>
       <h2>Meetings</h2>
-      <p>[[APPLICATION DESCRIPTION]]</p>
+      <p>This is the meetings plugin sample for the Webex JS SDK.</p>
+      <p>Each of the following sections within this sample perform specific actions that map to methods in the meetings plugin of the Webex JS SDK.</p>
+      <p>To find the exact logic performed by an action, please inspect the browser's developer console. We log in the console functions in this application that performs the selected action within the <code>./app.js</code> file.</p>
     </section>
 
     <section>
       <h2>Authentication</h2>
-      <p>[[AUTHENTICATION DESCRIPTION]]</p>
+      <p>Before using the meetings functionality of the Webex JS SDK, the Webex JS SDK must be initialized using a valid token. You may get test access token from our developer portal: <a href="https://developer.webex.com/docs/api/getting-started" target="_blank">https://developer.webex.com/docs/api/getting-started</a>. Once the JS SDK is has authenticated your token, it then must be registered as a device. Once these two steps have been performed, the meetings functionality of the Webex JS SDK should be available for usage.</p>
```

Comment:
Make it slightly more readable
```suggestion
      <p>Before using the meetings functionality of the Webex JS SDK, the Webex JS SDK must be initialized using a valid token.
      <p>You may get test access token from our developer portal: <a href="https://developer.webex.com/docs/api/getting-started" target="_blank">https://developer.webex.com/docs/api/getting-started</a>.</p>
      <p>Once the JS SDK is has authenticated your token, it then must be registered as a device. Once these two steps have been performed, the meetings functionality of the Webex JS SDK should be available for usage.</p>
```

Reply:
implemented.